exampleSite/config.toml

```diff
@@ -37,7 +37,10 @@ paginate = 10
 
     about_us = "<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>"
     copyright = "Copyright (c) 2015 - 2016, YourCompany; all rights reserved."
-    date_format = "2006-01-02"
```

Comment:
@ryanfox1985 in the config file it's already stated that the `date_format` has to be a Go's compatible time format, and an example is already given. So I don't see the need to add more complexity and check the date format in the templates.